### PR TITLE
Print out intermediate decoding errors

### PR DESCRIPTION
**Pull request by algorandskiy**
## Summary

Better errors on txn and dryrun objects decoding.

## Test Plan

Use existing tests

### CODE REVIEW

File: cmd/tealdbg/local.go [hunk 1]
```diff
@@ -128,6 +128,16 @@ func balanceRecordsFromParams(dp *DebugParams) (records []basics.BalanceRecord,
 		records = append(records, record)
 	}
 
+	// if conversion failed report all intermediate decoding errors
+	if err != nil {
+		if err1 != nil {
+			log.Printf("Decoding as JSON txn failed: %s", err1.Error())
```
Comment (tsachiherman) [inline cmd/tealdbg/local.go:134]: you're not decoding a txn here, you're decoding a balance record. 
i.e.
"Decoding blob as a JSON encoded single balance record failed : %s"

**Comment by algorandskiy** [comment]
Ny bad, wrong function

File: cmd/tealdbg/local.go (hunk 1 above)
Comment (algorandskiy) [reply cmd/tealdbg/local.go]: Fixed, thanks

**Review by tsachiherman — APPROVED** [review]
looks good to me.

**Pull request merged**